README.md

```diff
@@ -93,12 +93,19 @@ To run stress testing against the Salesforce instance, refer to the documentatio
 To get started working with our Phrase translations, you will need to:
 
 1. Install the Phrase CLI with `brew tap phrase/brewed && brew install phrase`
-1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in e.g. LastPass
+1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in Lastpass and as a local env var.
 
-**To upload new strings or translations to Phrase**
+- Each time you add new strings to locale-en.json, push to Phrase to keep it up-to-date
+- If you delete keys from locale-en.json, run phrase push, then run phrase cleanup to delete unused keys
+```
+phrase uploads cleanup --id=[upload id from the response of phrase push]  --access_token=[your access token]
+```
+- If you want to make updates directly to non-english languages, you need to update them in phrase, then pull in the latest values.
```

Comment:
```suggestion
   - If you want to make updates directly to non-english languages, you need to update them in phrase, then pull in the latest values.
```

nit: 3 spaces before the `-` will indent this like a sub-bullet. Posting this as a separate comment because the ticks on line 100/102 break up the suggestions